tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data; the ``/`` character in the dictionary keys allows pooling
+# across conditions by requesting partial condition descriptors (i.e.,
+# requesting ``'auditory'`` will select all epochs with Event IDs 1 and 2;
+# requesting ``'left'`` will select all epochs with Event IDs 1 and 3). An
+# example of this is shown in the next section. There is also a convenient
+# :func:`~mne.viz.plot_events` function for visualizing the distribution of
+# events across the duration of the recording (to make sure event detection
+# worked as expected):
+
+fig = mne.viz.plot_events(events, event_id=event_dict, sfreq=raw.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+###############################################################################
+# Epoching continuous data
+# ^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Scipy supports read and write of matlab files. You can save your single
-# trials with:
-
-from scipy import io  # noqa
-io.savemat('epochs_data.mat', dict(epochs_data=epochs_data), oned_as='row')
-
-##############################################################################
-# or if you want to keep all the information about the data you can save your
-# epochs in a fif file:
-
-epochs.save('sample-epo.fif', overwrite=True)
-
-##############################################################################
-# and read them later with:
-
-saved_epochs = mne.read_epochs('sample-epo.fif')
-
-##############################################################################
-# Compute evoked responses for auditory responses by averaging and plot it:
-
-evoked = epochs['aud_l'].average()
-print(evoked)
-evoked.plot(time_unit='s')
-
-##############################################################################
-# .. topic:: Exercise
+# The :class:`~mne.io.Raw` object and the events array are the bare minimum
+# needed to create an :class:`~mne.Epochs` object, which we create with the
+# :class:`mne.Epochs` class constructor. Here we'll also specify some data
+# quality constraints: we'll reject any epoch where peak-to-peak signal
+# amplitude is beyond reasonable limits for that channel type. This is done
+# with a *rejection dictionary*:
+
+reject_criteria = dict(mag=4e-12,     # 4000 fT
+                       grad=4e-10,    # 4000 fT/cm
+                       eeg=150e-6,    # 150 μV
+                       eog=250e-6)    # 250 μV
+
+###############################################################################
+# We'll also pass the event dictionary as the ``event_id`` parameter (so we can
+# work with easy-to-pool event labels instead of the integer event IDs), and
+# specify ``tmin`` and ``tmax`` (the time relative to each event at which to
+# start and end each epoch). Finally, since we didn't preload the
+# :class:`~mne.io.Raw` data, we'll tell the :class:`~mne.Epochs` constructor to
+# load the epoched data into memory:
```

Comment:
again, advanced stuff in my opinion. But you can decide.

Reply:
I'll keep `preload=True` here, and explain it's because we want to see the result of the `reject` dict right away (otherwise, its output doesn't happen until later, which I think will confuse folks).

Reply:
> otherwise, its output doesn't happen until later, which I think will confuse folks

yes absolutely agree!